cylc/flow/cfgspec/workflow.py

```diff
@@ -785,9 +785,22 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     :ref:`SequentialTasks`.
             ''')
 
+        Conf('sequential xtriggers default', VDR.V_BOOLEAN, False, desc='''
+            When set to ``True``, parentless tasks that trigger off xtriggers
+            will only spawn sequentially, i.e. on the satisfaction of the
+            xtriggers in order. Otherwise, these tasks will all spawn at the
+            same time up to the runahead limit.
+
+            This workflow-wide default can be overridden by a reserved
+            keyword argument in the xtrigger function declaration and/or
+            function (``sequential=True/False``).
+
+            The presence of one sequential xtrigger on a parentless task with
```

Comment:
Could be clearer.  A quick attempt at it:
```suggestion
            If ``True``, tasks that only depend on xtriggers will not spawn until their
            previous (cycle point) instance is satisfied. Otherwise, they will all spawn
            at once out to the runahead limit.

            This setting can be overridden by a reserved keyword argument in
            individual xtrigger declarations, or in xtrigger function definitions.

            One sequential xtrigger on a parentless task with
```

Reply:
Yes, far less clunky than my attempt.

> tasks that only depend on xtriggers

If xtriggers are the only non-graph prerequisites possible (external triggers?), you'd know :)

Reply:
> until their previous (cycle point) instance is satisfied.

Now that I read this again, it's not satisfaction of the previous (cycle point) instance, but the satisfaction of the sequential xtriggers of the previous (cycle point) instance.